workspaces/jupyterlab-python/install.sh

```diff
@@ -2,18 +2,42 @@
 
 set -ex
 
+# This script requires root.
+[ "$(id -u)" -eq 0 ] || exit 1
+
+# On PL, we want to standardize on using 1001:1001 for the user. We change
+# the "jovyan" account's default UID and GID here.
+USER_NAME=jovyan
+GROUP_NAME=$(id -gn $USER_NAME)
+OLD_UID=$(id -u $USER_NAME)
+OLD_GID=$(id -g $USER_NAME)
+NEW_UID=1001
+NEW_GID=1001
+groupmod -g $NEW_GID $GROUP_NAME
+usermod -u $NEW_UID -g $NEW_GID $USER_NAME
+for d in /home /tmp ; do
+    find $d -user $OLD_UID -execdir chown -h $NEW_UID {} +
+    find $d -group $OLD_GID -execdir chgrp -h $NEW_GID {} +
+done
+
 # Install dependencies and various libraries
 apt-get update && apt-get -y upgrade
-apt-get -y install gcc graphviz graphviz-dev
+apt-get -y install gosu gcc graphviz graphviz-dev
+
+# Test gosu and prepare the gosu helper.
```

Comment:
Nit: what does it mean to "test gosu"?

Reply:
The gosu installation instructions [demonstrated doing this](https://github.com/tianon/gosu/blob/master/INSTALL.md) to check that it's working as intended (although it doesn't seem comprehensive).

Reply:
This feels a little idiosyncratic; none of our other Dockerfiles test things that they install, AFAIK. It's your call whether you think this is valuable enough to leave here.

Reply:
I just removed it. It would have made more sense to test whether `jovyan` had the expected uid:gid but it's an arbitrary thing to unit test out of all the other things in the install process.